### PR TITLE
Add feature to disable hotReloading with query string

### PR DESCRIPTION
**Pull request by edi9999**
**What kind of change does this PR introduce?**
Feature

**Did you add or update the `examples/`?**
No, but you could use any sample and use the queryParam : `?disableAutoreload=true` to 

**Summary**

See corresponding issue : https://github.com/webpack/webpack-dev-server/issues/936

This option can be used if you want to use the same platform for developping and to run automated tests (the tests should not be impacted by a code change whenever they are running). 

**Does this PR introduce a breaking change?**

No

**Other information**

### CODE REVIEW

**Comment by edi9999** [comment]
Any updates ?
ping @shellscape

**Comment by shellscape** [comment]
@edi9999 please do us a solid and wait about a week before pings 😄 I promise we see all activity here and will respond/comment as soon as possible.  I'm currently preparing for hurricane irma and will be returning next week.

**Comment by edi9999** [comment]
Ok ! I hope everything will be OK for you

File: client/index.js [hunk 1]
```diff
@@ -22,6 +22,10 @@ function getCurrentScriptSource() {
 }
 
 let urlParts;
+let hotReloading = true;
+if (typeof window !== 'undefined') {
+  hotReloading = window.location.search.indexOf('disableAutoReload') === -1;
```
Comment (shellscape) [inline client/index.js]: not a fan of negative-options, meaning a positive value will result in the negative reassignment of another variable. let's also keep the variable names the same as the querystring parameter. so;

```js
let hotReload = true;
...
const qs = window.location.search.toLowerCase();
hotReload = !qs.indexOf('hotreload=false');
```

the code was trimmed for brevity, but those are the changes I'd like to see

Comment (edi9999) [reply client/index.js]: Fixed :-)

**Pull request merged**